lib/components/segmented-control/segmented-control.scss

```diff
@@ -0,0 +1,105 @@
+@import "../../styles/index.scss";
+
+.segmented-control {
+    display: flex;
+    align-items: center;
+    width: fit-content;
+    background: color-mix(
+        in srgb,
+        var(--semantic-color-typography-prominent) 4%,
+        transparent
+    );
+
+    .item {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+
+        &.disabled {
+            opacity: var(--core-opacity-300);
+            cursor: not-allowed;
+            pointer-events: none;
+        }
+        &:hover:not(.disabled) {
+            background: color-mix(
+                in srgb,
+                var(--semantic-color-typography-prominent) 8%,
+                transparent
+            );
```

Comment:
Not sure if we really need `color-mix` can you check if this can be achieved using ` --core-color-opacity-black-100`?

Reply:
@prince-deriv in this case, it won't work for dark theme